### PR TITLE
feat: Traditional Chinese language support

### PR DESCRIPTION
**Pull request by aykhara**
Traditional Chinese translation verified by @rextangtw

### CODE REVIEW

File: src/common/localization/zh-tw.ts [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+                description: "一個或多個區域尚未被標記。在繼續下一個圖像數據之前，請確保所有區域均已標記。",
+                // 1 or more regions have not been tagged.
+                // Ensure all regions are tagged before continuing to next asset.
+            },
+        },
+    },
+    export: {
+        title: "匯出", // Export
+        settings: "匯出設定", // Export Settings
+        saveSettings: "保存匯出設定", // Save Export Settings
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
        saveSettings: "儲存匯出設定", // Save Export Settings
```

File: src/common/localization/zh-tw.ts [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+                        description: "是否在目標連接中包括二進位圖像數據",
+                        // Whether or not to include binary image assets in target connection
+                    },
+                },
+            },
+            vottJson: {
+                displayName: "VoTT JSON", // VoTT JSON
+            },
+            azureCV: {
+                displayName: "Azure客製化視覺服務", // Azure Custom Vision Service
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
                displayName: "Azure自訂視覺服務", // Azure Custom Vision Service
```

File: src/common/localization/zh-tw.ts [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+                    westUs2: "美國西部 2", // West US 2
+                    westEurope: "西歐", // West Europe
+                },
+                properties: {
+                    apiKey: {
+                        title: "API密鑰", // API Key
+                    },
+                    region: {
+                        title: "區域", // Region
+                        description: "部署服務的Azure地區", // The Azure region where your service is deployed
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
                        description: "部署服務的Azure區域", // The Azure region where your service is deployed
```

File: src/common/localization/zh-tw.ts [hunk 4]
```diff
@@ -0,0 +1,514 @@
...
+                        title: "專案名", // Project Name
+                    },
+                    description: {
+                        title: "專案簡介", // Project Description
+                    },
+                    domainId: {
+                        title: "領域", // Domain
+                    },
+                    newOrExisting: {
+                        title: "新專案或既有專案", // New or Existing Project
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
                        title: "新增專案或既有專案", // New or Existing Project
```

File: src/common/localization/zh-tw.ts [hunk 5]
```diff
@@ -0,0 +1,514 @@
...
+                        options: {
+                            new: "新專案", // New Project
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
                            new: "新增專案", // New Project
```

File: src/common/localization/zh-tw.ts [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+            },
+            cntk: {
+                displayName: "Microsoft Cognitive Toolkit（CNTK)", // Microsoft Cognitive Toolkit (CNTK)
+            },
+            csv: {
+                displayName: "逗號分隔格式 (CSV)", // Comma Separated Values (CSV)
+            },
+        },
+        messages: {
+            saveSuccess: "已成功保存匯出設定", // Successfully saved export settings
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
            saveSuccess: "已成功儲存匯出設定", // Successfully saved export settings
```

File: src/common/localization/zh-tw.ts [hunk 7]
```diff
@@ -0,0 +1,514 @@
...
+                    },
+                },
+                autoDetect: {
+                    title: "自動偵測", // Auto Detect
+                    description: "在圖像數據之間瀏覽時是否自動進行預測",
+                    // Whether or not to automatically make predictions as you navigate between assets
+                },
+                modelPath: {
+                    title: "模型路徑", // Model path
+                    description: "從本地檔案系統中選擇模型", // Select a model from your local file system
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
                    description: "從本機檔案系統中選擇模型", // Select a model from your local file system
```

File: src/common/localization/zh-tw.ts [hunk 8]
```diff
@@ -0,0 +1,514 @@
...
+                predictTag: {
+                    title: "預測標記", // Predict Tag
+                    description: "是否在預測中自動包含標記", // Whether or not to automatically include tags in predictions
+                },
+            },
+        },
+        messages: {
+            loadingModel: "正在載入主動學習模型...", // Loading active learning model...
+            errorLoadModel: "載入主動學習模型時出現錯誤", // Error loading active learning model
+            saveSuccess: "已成功保存主動學習設定", // Successfully saved active learning settings
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
            saveSuccess: "已成功儲存主動學習設定", // Successfully saved active learning settings
```

File: src/common/localization/zh-tw.ts [hunk 9]
```diff
@@ -0,0 +1,514 @@
...
+            message: "該應用程式遇到未知錯誤。請再試一遍。", // The app encountered an unknown error. Please try again.
+        },
+        projectUploadError: {
+            title: "上傳檔案時出現錯誤", // Error Uploading File
+            message: "上傳檔案時出現錯誤。請確認檔案格式正確，然後重試。",
+            // There was an error uploading the file. Please verify the file is of the correct format and try again.
+        },
+        genericRenderError: {
+            title: "載入應用程序時出現錯誤", // Error Loading Application
+            message: "描繪應用程序時發生錯誤。請再試一遍", // An error occured while rendering the application. Please try again
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
            message: "轉譯應用程序時發生錯誤。請再試一遍", // An error occured while rendering the application. Please try again
```

File: src/common/localization/zh-tw.ts [hunk 10]
```diff
@@ -0,0 +1,514 @@
...
+            // The selected project files does not contain valid JSON. Please check the file any try again.
+        },
+        projectDeleteError: {
+            title: "刪除專案時出現錯誤", // Error deleting project
+            message: "刪除專案時發生錯誤。請確認專案檔案和安全性認證是否存在，然後重試",
+            // An error occured while deleting the project.
+            // Validate the project file and security token exist and try again
+        },
+        securityTokenNotFound: {
+            title: "加載專案檔案時出現錯誤", // Error loading project file
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
            title: "載入專案檔案時出現錯誤", // Error loading project file
```

File: src/common/localization/zh-tw.ts [hunk 11]
```diff
@@ -0,0 +1,514 @@
...
+            // There was an error loading the canvas, check the project's assets and try again.
+        },
+        importError: {
+            title: "匯入V1格式專案時出現錯誤", // Error importing V1 project
+            message: "匯入V1格式專案時出現錯誤。請檢查專案檔案，然後重試",
+            // There was an error importing the V1 project. Check the project file and try again
+        },
+        pasteRegionTooBigError: {
+            title: "貼上區域時發生錯誤", // Error pasting region
+            message: "此區域對於本圖像數據來說太大了。請嘗試複製其他的區域",
```
Comment (ShuyuanChang) [inline src/common/localization/zh-tw.ts]:
```suggestion
            message: "此區域對於這一個圖像數據太大了。請嘗試複製其他的區域",
```

File: src/common/localization/zh-tw.ts [hunk 12]
```diff
@@ -0,0 +1,514 @@
...
+            selectConnection: "選擇連線", // Select a Connection
+        },
+        recentProjects: "最近的專案", // Recent Projects
+        deleteProject: {
+            title: "刪除專案", // Delete Project
+            confirmation: "確定要刪除專案?", // Are you sure you want to delete project
+        },
+        importProject: {
+            title: "匯入專案", // Import Project
+            confirmation: "您確定要將專案$ {project.file.name}的設定轉換為v2格式嗎？我們建議您首先備份專案文件。",
```
Comment (mydiemho) [inline src/common/localization/zh-tw.ts]: please remove the space between `$` and `{project.file.name}`.

File: src/common/localization/zh-tw.ts [hunk 13]
```diff
@@ -0,0 +1,514 @@
...
+            // Are you sure you want to convert project ${project.file.name} project settings to v2 format?
+            // We recommend you backup the project file first.
+        },
+        messages: {
+            deleteSuccess: "已成功刪除$ {project.name}專案", // Successfully deleted ${project.name}
```
Comment (mydiemho) [inline src/common/localization/zh-tw.ts]: same

File: src/common/localization/zh-tw.ts [hunk 14]
```diff
@@ -0,0 +1,514 @@
...
+            description: "在哪裡保存專案和匯出的資料", // Where to save the project and exported data
+        },
+        videoSettings: {
+            title: "影片設定", // Video Settings
+            description: "設定影片標記的速率", // The rate at which frames are extracted for tagging.
+            frameExtractionRate: "影像取樣率（影像每秒的畫面數）", // Frame Extraction Rate (frames per a video second)
+        },
+        addConnection: "新增連線", // Add Connection
+        messages: {
+            saveSuccess: "已成功保存$ {project.name}專案設定", // Successfully saved ${project.name} project settings
```
Comment (mydiemho) [inline src/common/localization/zh-tw.ts]: same

File: src/common/localization/zh-tw.ts [hunk 15]
```diff
@@ -0,0 +1,514 @@
...
+        },
+    },
+    projectMetrics: {
+        title: "專案相關指標", // Project Metrics
+        assetsSectionTitle: "圖像數據", // Assets
+        // As for this VOTT tool, translate "Assets" to "Image data" in Traditional Chinese,
+        // as "Asset" can be confusing if directly translated.
+        totalAssetCount: "圖像數據總數", // Total Assets
+        visitedAssets: "已檢視的圖像數據（$ {count}）", // Visited Assets (${count})
```
Comment (mydiemho) [inline src/common/localization/zh-tw.ts]: same

**Comment by rextangtw** [comment]
> Please build a version locally and make sure all strings are rendered correctly. In the process of translation, a lot of the variables extrapolations are messed up. I mark some below but not all please double check

thanks @mydiemho , I just updated 9 occurrence of those and committed.

File: src/common/localization/zh-tw.ts (hunk 12 above)
Comment (rextangtw) [reply src/common/localization/zh-tw.ts]: updated

File: src/common/localization/zh-tw.ts (hunk 13 above)
Comment (rextangtw) [reply src/common/localization/zh-tw.ts]: updated

File: src/common/localization/zh-tw.ts (hunk 14 above)
Comment (rextangtw) [reply src/common/localization/zh-tw.ts]: updated

File: src/common/localization/zh-tw.ts (hunk 15 above)
Comment (rextangtw) [reply src/common/localization/zh-tw.ts]: updated

**Pull request merged**